protocol/ignored-changes.xml

```diff
@@ -36,4 +36,14 @@
     <differenceType>6001</differenceType>
     <field>DELETED</field>
   </difference>
+  <difference>
+    <className>io/zeebe/protocol/record/intent/MessageSubscriptionIntent</className>
```

Comment:
Can we maybe just ignore intent here as well? like via *

Reply:
Yes. Since we will ignore Clirr, I don't case :sweat_smile: